test/luatest_helpers/server.lua

```diff
@@ -160,6 +160,16 @@ function Server:instance_uuid()
     return uuid
 end
 
+function Server:replicaset_uuid()
+    -- Cache the value when found it first time.
+    if self.replicaset_uuid_value then
```

Comment:
Could line 168 return box.NULL somehow? Maybe explicit check to nil is better.

Reply:
I did this:
```Diff
@@ -166,6 +166,11 @@ function Server:replicaset_uuid()
         return self.replicaset_uuid_value
     end
     local uuid = self:exec(function() return box.info.cluster.uuid end)
+    if uuid == nil then
+        -- Probably didn't bootstrap yet. Listen starts before replicaset UUID
+        -- is assigned.
+        return nil
+    end
     self.replicaset_uuid_value = uuid
     return uuid
 end
```